### ISSUE TITLE
Dimensional rift broken again

### ISSUE THREAD

**Issue by jknpj** [d3athrow/vgstation13#12818]
#### Description of issue

![literally why](https://cloud.githubusercontent.com/assets/17928298/21249018/e30201b8-c319-11e6-81a1-46be6b68d55f.png)

Dimensional rift does not work at all

#### Difference between expected and actual behavior

Expected: Escape to the metaclub's beach
Actual: Nothing

#### Steps to reproduce 

Be a madmen and blow a crystal, now try fight your destiny

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by Iamgoofball** [comment]
at least it's not Mr. Clean's head

**Comment by Exxion** [comment]
How can people consistently fail to write the checkboxes properly? It says exactly how to do it right above them.

**Comment by jknpj** [comment]
Still a thing

**Comment by SysadminJeroen** [comment]
Has anyone even tried to fix this?

**Comment by SysadminJeroen** [comment]
Does anyone know when it still worked, if that matters.

**Comment by SysadminJeroen** [comment]
Started bisecting what broke this.

Will update this comment on each step.

```sh
git bisect start
# bad: [abc2b7c8c7bcd358fa1852877087de0851739e08] Smooth Dark Shuttle Walls (#12815)
git bisect bad abc2b7c8c7bcd358fa1852877087de0851739e08
# good: [2afb3740401d6e7f000a1da52afe3c9b84b1c711] Automatic changelog update.
git bisect good 2afb3740401d6e7f000a1da52afe3c9b84b1c711
# good: [0cba97aaec8a91748425db50687636b3dc694b3b] Automatic changelog update.
git bisect good 0cba97aaec8a91748425db50687636b3dc694b3b
# bad: [f5b047cc3bd964de3bae3ea5538842eba108f7a3] Adds IC in OOC protection to LOOC. (#12228)
git bisect bad f5b047cc3bd964de3bae3ea5538842eba108f7a3
# bad: [b656f91a1c4eef7256c1feeddc875b2cd0382d0d] Ring of fire spellbook (#11872)
git bisect bad b656f91a1c4eef7256c1feeddc875b2cd0382d0d
# good: [42306dec4b16290f60c47d1e83dfeaeb5c23086f] Spellbook refactor (#11510)
git bisect good 42306dec4b16290f60c47d1e83dfeaeb5c23086f
# good: [7eecf52ba56b39f61030044fc29c838347cead39] Bugfixes (#11779)
git bisect good 7eecf52ba56b39f61030044fc29c838347cead39
# good: [d39ae64b0371645b5a422c0165b98787cc0e6435] Better coat rack (#11785)
git bisect good d39ae64b0371645b5a422c0165b98787cc0e6435
# bad: [69000c99c9df510ccaa373da7a79d5414f29f0fb] Automatic changelog update.
git bisect bad 69000c99c9df510ccaa373da7a79d5414f29f0fb
# bad: [d5f775407df6f131b5b917b6a30b29488d34a8b9] Fixes dropping items through portals (#11841)
git bisect bad d5f775407df6f131b5b917b6a30b29488d34a8b9
# bad: [3b45f4804e1a9bc86ee1afd78f41d4b083d0b44e] Fixing some bugs. (#11829)
git bisect bad 3b45f4804e1a9bc86ee1afd78f41d4b083d0b44e
# good: [970c6f0686e815f0f7da30b35ca54ed17adc3a79] Automatic changelog update.
git bisect good 970c6f0686e815f0f7da30b35ca54ed17adc3a79
# first bad commit: [3b45f4804e1a9bc86ee1afd78f41d4b083d0b44e] Fixing some bugs. (#11829)
```

**Comment by SysadminJeroen** [comment]
Finished a successful bisect of the bug.

The commit 3b45f4804e1a9bc86ee1afd78f41d4b083d0b44e introduced the bug.

Log of the bisect is available here: https://github.com/vgstation-coders/vgstation13/issues/12818#issuecomment-314229875
